crates/base/src/datetimes.rs

```diff
@@ -19,20 +22,42 @@ pub struct HMS {
 
 pub struct YMDHMS(pub i16, pub u8, pub u8, pub u8, pub u8, pub u8);
 
+/// The default timezone is specified at the server's startup stage.
+pub static DEFAULT_TIMEZONE: SyncOnceCell<TimeZone> = SyncOnceCell::new();
```

Comment:
this works at first eye. But when you make functions working on columns with its own timezone, global setting here fails. 

The correct way is that we extend these functions in this module with tz arguments,  which passed from Arrow calls.